ci/install_data_services.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+
+import sys
+
+import sdk_install, sdk_cmd
+import logging
+
+logging.basicConfig(level=logging.INFO)
+log = logging.getLogger(__name__)
+
+# TODO get_cluster_name and check_authentication functions in duplicate from bundle_bootstrap.py
+#      this functions MUST be moved in one place after refactoring
```

Comment:
I'm wondering why we even need to call these functions here? This is supposed to be simple internal glue code for CI, just run `sdk_install.install` and make sure the error is propagated, there is no end user whose hand we need to hold here :-)\

Reply:
additional functions removed

Reply:
during non authorized `dcos` run getting a few mins loop in the output of:
```html
<html>
<head>
  <title>Unauthorized</title>
  <noscript><meta http-equiv="refresh" content="5; url=/#/login"></noscript>
  <script>
    (function () {
      var location = window.location;
      location.href = "/#/login?redirect=" + encodeURIComponent(location.href);
    }())
  </script>
</head>
<body>
  <h1>Unauthorized</h1>
</body>
</html>
```